references.bib

```diff
@@ -16,3 +16,22 @@ @book{algebraic.topology-by:hatcher
   addendum={Page numbers are based on the online free version},
   address={Cambridge, UK}
 }
+
+@InProceedings{sterling_et_al:LIPIcs:2019:10538,
+  author =	{Jonathan Sterling and Carlo Angiuli and Daniel Gratzer},
+  title =	{{Cubical Syntax for Reflection-Free Extensional Equality}},
+  booktitle =	{4th International Conference on Formal Structures for Computation and Deduction (FSCD 2019)},
+  pages =	{31:1--31:25},
+  series =	{Leibniz International Proceedings in Informatics (LIPIcs)},
+  ISBN =	{978-3-95977-107-8},
+  ISSN =	{1868-8969},
+  year =	{2019},
+  volume =	{131},
+  editor =	{Herman Geuvers},
+  publisher =	{Schloss Dagstuhl--Leibniz-Zentrum fuer Informatik},
+  address =	{Dagstuhl, Germany},
+  URL =		{http://drops.dagstuhl.de/opus/volltexte/2019/10538},
+  URN =		{urn:nbn:de:0030-drops-105387},
+  doi =		{10.4230/LIPIcs.FSCD.2019.31},
```

Comment:
There's no need to have URL once you have DOI. :-)

Reply:
Should I remove it? I just copy-pasted the BibTeX from the link you sent :)

Reply:
Yes. Usually you need to edit the provided bib items on the Internet.